src/socketio.quiver.js

```diff
@@ -115,23 +115,23 @@ QuiverSocialProvider.DEFAULT_SERVERS_ = [{
 }, {
   type: 'socketio',
   domain: 'dzgea1sj9ik08.cloudfront.net',
-  front: 'sdk.amazonaws.com'
+  front: 'js.arcgis.com'
 }, {
   type: 'socketio',
   domain: 'd2oi2yhmhpjpt7.cloudfront.net',
   front: 'api.mapbox.com'
 }, {
   type: 'socketio',
   domain: 'd2cqtpyb8m6x8r.cloudfront.net',
-  front: 'cdn.tinymce.com'
+  front: 'cdn.ckeditor.com'
 }, {
   type: 'socketio',
   domain: 'd3h805atiromvi.cloudfront.net',
   front: 'assets.tumblr.com'
 }, {
   type: 'socketio',
   domain: 'd2yp1zilrgqkqt.cloudfront.net',
-  front: 'www.splunk.com'
+  front: 'cdn.livefyre.com'
 }];
```

Comment:
@bemasc Nice! Also I'd love to know how you found and decided on these! Can the process be automated, at least partially (and if so would it be worth it)?